test/test_unary_ufuncs.py

```diff
@@ -450,6 +450,38 @@ def test_out_arg_all_dtypes(self, device, dtype, op):
             else:
                 self._test_out_arg(op, input, out)
 
+    @ops(filter(lambda op: op.supports_dtype_kwarg, unary_ufuncs), dtypes=OpDTypes.supported)
+    def test_dtype_arg(self, device, dtype, op):
+        input = make_tensor((32, 32), dtype=dtype, device=device,
+                            low=op.domain[0], high=op.domain[1])
+
+        for computation_dtype in all_types_and_complex_and(torch.bool, torch.half):
```

Comment:
The computation_dtype's should probably be derived from the op's supported dtypes, which can be queried for from the OpInfo

Reply:
Yes.

Reply:
Done.